lib/galaxy/workflow/run_request.py

```diff
@@ -436,6 +453,9 @@ def workflow_request_to_run_config(work_request_context, workflow_invocation):
                 use_cached_job = (parameter.value == 'true')
         elif parameter_type == param_types.RESOURCE_PARAMETERS:
             resource_params[parameter.name] = parameter.value
+        elif parameter_type == param_types.STEP_PARAMETERS:
+            import json
```

Comment:
json is alredy imported at this point, right ?

Reply:
Yeah - made this change while "polishing" PR yesterday. I should have reviewed more closely - this is kind of a mess. Thanks - I'll fix this also.